docs/integrations/tools/llmlingua-prompt-compression-tool.md

```diff
@@ -0,0 +1,42 @@
+# LLMLingua Prompt Compression
```

Comment:
need change the index.md file

![image](https://github.com/microsoft/promptflow/assets/47586720/76926d2c-b981-46b4-9959-1e764f8ce462)